keep/api/models/db/migrations/versions/2024-09-01-14-04_94886bc59c11.py

```diff
@@ -0,0 +1,47 @@
+"""user_name and generated_name separation for incident model added
+
+Revision ID: 94886bc59c11
+Revises: 1c650a429672
+Create Date: 2024-09-01 14:04:52.407708
+
+"""
+
+import sqlalchemy as sa
+import sqlmodel
+from alembic import op
+
+# revision identifiers, used by Alembic.
+revision = "94886bc59c11"
+down_revision = "1c650a429672"
+branch_labels = None
+depends_on = None
+
+
+def upgrade() -> None:
+    # ### commands auto generated by Alembic - please adjust! ###
+    with op.batch_alter_table("incident", schema=None) as batch_op:
+        batch_op.add_column(
+            sa.Column("user_name", sqlmodel.sql.sqltypes.AutoString(), nullable=True)
```

Comment:
column "user_name" on incident table sounds like the user who created the incident. I would change to `user_generated_name` and `ai_generated_name`